ui/src/shared/constants/index.js

```diff
@@ -468,6 +468,8 @@ export const STROKE_WIDTH = {
   light: 1.5,
 }
 
+export const HEARTBEAT_INTERVAL = 10000 // ms
```

Comment:
It's a great idea, but unfortunately JavaScript has no access to the cookies. One of the downsides of cookies.